packages/core/src/types/hasher.ts

```diff
@@ -22,4 +22,17 @@ export abstract class IHasher {
   public isElementSizeValid = (element: HexString): boolean => IHasher.byteSize(element) <= this.options.blockSizeBits;
 
   static byteSize = (str: string) => new Blob([str.startsWith("0x") ? str.slice(2) : str]).size;
+
+  hashSingle(data: string): string {
+    return this.hash([data]);
+  }
+
+  public getGenesis = (): HexString => {
+    const s = "brave new world";
```

Comment:
Can we make `s` a constant and write an associated comment explaining the reasoning behind?

Reply:
(A global constant, not referring to a local constant as it's already one)